### PR TITLE
Store image urls in datastore

### PR DESCRIPTION
**Pull request by AshleyLTW**
I modified postUserReport in script.js so that it sends a post request to the url returned by fetchBlobstoreURL. In ReportServlet.java I upload the image to a url and store that url in datastore. 

For ease of testing, I've also set the report form to be always visible, will revert this once I can display images from blobstore.

### CODE REVIEW

File: web-app/src/main/java/com/google/sps/servlets/BlobstoreUploadUrlServlet.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ * of the response is the URL that allows a user to upload a file to Blobstore.
+ * If this sounds confusing, try running a dev server and navigating to
+ * /blobstore-upload-url to see the Blobstore URL.
+ */
+@WebServlet("/blobstore-upload-url")
+public class BlobstoreUploadUrlServlet extends HttpServlet {
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        System.out.println("java blobstore url accessed");
```
Comment (jiminbyun) [inline web-app/src/main/java/com/google/sps/servlets/BlobstoreUploadUrlServlet.java]: nit: Debugging message not needed anymore

File: web-app/src/main/java/com/google/sps/servlets/ReportServlet.java [hunk 2]
```diff
@@ -70,6 +78,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
             response.getWriter().println(exception);
           }
           break;
+        case "image":
+          reportEntity.setProperty(paramName, getUploadedFileUrl(request, "image"));
```
Comment (jiminbyun) [inline web-app/src/main/java/com/google/sps/servlets/ReportServlet.java]: If you are not using value here, maybe it doesn't needs to be inside the loop & switch statement

File: web-app/src/main/java/com/google/sps/servlets/ReportServlet.java [hunk 3]
```diff
@@ -32,14 +38,17 @@
...
+    {
       put("title", "");
       put("latitude", "0.0");
       put("longitude", "0.0");
       put("timestamp", "0");
       put("incidentType", "etc");
-      put("description", ""); 
-  }};
+      put("description", "");
+      put("image", "");
```
Comment (jiminbyun) [inline web-app/src/main/java/com/google/sps/servlets/ReportServlet.java]: I think you deal with null case seperately with getUploadedFileUrl function, so there is no need to add default value here

**Review by jiminbyun — APPROVED** [review]

File: web-app/src/main/webapp/script.js [hunk 4]
```diff
@@ -57,7 +57,7 @@ function showMessageOnInfoWindow(message, position, map, infoWindow) {
 }
 
 window.onload = () => {
-  document.getElementById("report-form").style.display = "none";
+  // document.getElementById("report-form").style.display = "none";
```
Comment (darrnshn) [inline web-app/src/main/webapp/script.js]: nit: is this intentional? Either uncomment this or remove it completely.

**Review by darrnshn — APPROVED** [review]

**Pull request merged**